core/embed/rust/src/ui/util.rs

```diff
@@ -64,14 +65,13 @@ pub fn set_animation_disabled(_disabled: bool) {}
 /// Display an icon and a text centered relative to given `Point`.
 pub fn icon_text_center(
     baseline: Point,
-    icon: &'static [u8],
+    icon: &Icon,
     space: i16,
     text: &str,
     style: TextStyle,
     text_offset: Offset,
 ) {
-    let toif_info = unwrap!(display::toif_info(icon), "Invalid TOIF data");
-    let icon_width = toif_info.0.y;
+    let icon_width = icon.width();
```

Comment:
hmm there is a large UI diff because i incidentally fixed this bug affecting homescreen notification positioning
![download1](https://user-images.githubusercontent.com/37884576/210437412-f9222f99-ae63-45cd-8819-be03e85bdcf7.png) ![download2](https://user-images.githubusercontent.com/37884576/210437434-6a0430ce-b5fd-4aac-981b-1904831319da.png) ![diff](https://user-images.githubusercontent.com/37884576/210437472-607ad66b-f190-4313-8c00-555753267e76.png)

Reply:
updated the fixtures: https://github.com/trezor/trezor-firmware/pull/2708/commits/da5e028c772dd06e8e041ba4953ec5f0ccf844c5